stage1/00-boot-files/files/config.txt

```diff
@@ -54,11 +54,16 @@ disable_splash=1
 
 # Enable audio (loads snd_bcm2835)
 dtparam=audio=on
-gpu_mem=256
-[pi4]
-# Enable DRM VC4 V3D driver on top of the dispmanx display stack
-dtoverlay=vc4-fkms-v3d
-max_framebuffers=2
+
+# Enable UART
+# Note: on boards with the wireless/Bluetooth, this fixes the core frequency to 250MHz, unless force_turbo is set.
+#enable_uart=1
```

Comment:
Hm, so I might miss something, but why is this commented out?

Reply:
That is how it is by default on RPi. I made a compromise towards performance here, we will include a note on this in the documentation. Enabling UART by default requires to either sacrifice the performance or force turbo (needs good cooling). On Pyxis hardware, UART will be enabled by default: there is no BT module and we will use the full asynchronously-clocked UART there. See here, https://www.raspberrypi.org/documentation/configuration/uart.md